src/types.ts

```diff
@@ -200,3 +200,5 @@ interface PlayerItems {
 	href: string
 	version: string
 }
+
+export type StoreType = 'credits' | 'marks'
```

Comment:
Suggestion:

```ts
export const STORE_TYPES = ['credits', 'marks'] as const
export type StoreType = typeof STORE_TYPES[number]
```

That way when you're constructing the `<select>` you can map STORE_TYPES to `<option>`s and the options & type will always be in sync

Reply:
Great idea, will implement shortly